### ISSUE TITLE
New typeface in logo

### ISSUE THREAD

**Issue by dandaka** [OpenLightingProject/logos#9]
OLA logo uses Century Gothic typeface. It looks fine in "OLA". It is not that good in "Open Lighting Project". Seems that original design had only "OLA" case, which explains not perfect performance in other cases.

I have two candidates, that look similar, but are better in long lines and have a more contemporary look.

I have included the original typeface, so you could compare. Please note g, t, j letters — they look kinda ugly in Century Gothic and much better with new typefaces.

![ola-logo-ideas](https://cloud.githubusercontent.com/assets/2384694/12704079/b855b218-c863-11e5-8d18-93e0691ca7af.png)

Let me know what do you think about it.

**Comment by nomis52** [comment]
I do like how the right edge of the L is parallel to the A in the existing logo. 

Also the existing OLA logo is trademarked, I'm not sure how much we can modify it before we need to reapply.

**Comment by nomis52** [comment]
If I had to pick one I'd go with Cera Pro Regular . Campton looks too bold.

**Comment by DouglasHeriot** [comment]
You are correct the "Open Lighting Project" design was an afterthought. I would not replace the "OLA" mark with any of those – the kerning is much sloppier on all of them. I put a bit of effort into getting the OLA to sit nicely together.

However, I agree the font Century Gothic is not that great for "Open Lighting Project" (or not that great in general). I would remove the 5-pin dots from its ‘O’, and keep it separate from the OLA mark. It can use a different typeface (one of your suggested ones), which complement the existing OLA mark.

Also, I’m curious what the licensing is on the fonts Cera and Campton. Both of them are commercial fonts, that seem to have some sort of free download. I have not been able to find licensing information if we were to use it as a webfont for copy in a web page. It would be good to choose a font that is a reasonable price (or free), and has reasonable web licensing options.

**Comment by dandaka** [comment]
1) I have replaced Campton Book with Campton Light to match thickness

![ola-logo-ideas](https://cloud.githubusercontent.com/assets/2384694/12712859/6a5dcc84-c8dc-11e5-9866-ff7a4aaa40e1.png)

2) This is only a typeface proposition. Custom editing (like L shape) and kerning could and should be done, if there is an interest.

3) If we decide to change typeface in "long" version, I think its logical to tune short "OLA" mark also.

4) I think 5-pin dots are giving this typeface an unique look and should be preserved.

5) I don't think we need to use these as webfonts. For webfonts we could find some good free fonts, that will complement fonts in logo.

As for licence for a logo, I believe Campton is 30$ and Cera is 55$.
